### DISCUSSION TITLE
Changing the scale for stacked plots

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#3835] [Q&A] [answered]
Hi All,

Could someone point me to the PyAEDT method/s that will help to change the scales of the individual plots for a stacked plot display?

Thanks in advance.

Regards

**Comment by Samuelopez-ansys** [comment]
@Abhi-SimuTech Please give more information.

First, is it for AEDT plot or matplotlib? Because for complex postprocessing I would use matplotlib.

Second, could you provide a snipped code to reproduce the stacked plot?

Please try to describe all the steps needed because PyAEDT covers a lot of different applications, and the community can not help with such general questions.

Thanks

**Comment by Samuelopez-ansys** [comment, marked as answer]
By the way, if you try to change the scale of 2 stacked plot, it is not possible, I found a bug in AEDT and I already submitted it. Your colleague Steve Baker asked the same internally, please try to do not duplicate requests.

As a workaround you can use matplotlib, where you can define any kind of plot with AEDT data:

https://matplotlib.org/stable/gallery/subplots_axes_and_figures/subplots_demo.html
